fdbclient/admin_client.go

```diff
@@ -529,6 +529,16 @@ func (client *cliAdminClient) ChangeCoordinators(addresses []fdbv1beta2.ProcessA
 
 // GetConnectionString fetches the latest connection string.
 func (client *cliAdminClient) GetConnectionString() (string, error) {
+	adminClientMutex.Lock()
+	defer adminClientMutex.Unlock()
+
+	version, err := fdbv1beta2.ParseFdbVersion(client.Cluster.Spec.Version)
```

Comment:
We can remove that version check, we don't do that in other places when the functionality is available in all supported versions.